### PR TITLE
Remove server info

### PR DESCRIPTION
**Pull request by licitdev**
Remove unnecessary disclosure of server info.
Merge from https://github.com/directus/directus/pull/15506.

### CODE REVIEW

**Review by paescuj — APPROVED** [review]
LGTM 🚀

**Review by rijkvanzanten — APPROVED** [review]

File: .changeset/giant-ligers-nail.md [hunk 1]
```diff
@@ -4,4 +4,4 @@
 "docs": patch
 ---
 
-Removed server info
+Removed unnecessary disclosure of server info
```
Comment (rijkvanzanten) [inline .changeset/giant-ligers-nail.md:7]:
```suggestion
Removed unnecessary disclosure of potentially sensitive server info
```

**Comment by wc-matteo** [comment]
we used the version number return by that endpoint. Is there any way to read the directus version via api now? @licitdev

**Comment by rijkvanzanten** [comment]
> we used the version number returned by that endpoint. Is there any way to read the directus version via api now? @licitdev

I'd recommend creating a custom endpoint extension that returns that information at your own risk 👍🏻

**Pull request merged**